src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -169,7 +169,7 @@ public void should_return_a_200_with_search_results_based_on_state_assigned() {
     @Test
     public void should_return_a_200_with_search_results_with_restricted_role_assignment() {
         TestVariables taskVariablesSscs =
-            common.setupTaskAndRetrieveIdsWithCustomVariable(CamundaVariableDefinition.JURISDICTION, "SSCS");
+            common.setupTaskAndRetrieveIdsWithCustomVariable(CamundaVariableDefinition.JURISDICTION, "IA");
```

Comment:
This shouldn't be changed this test is testing restricted role-assignments which should grant you access to specific cases regardless of the jurisdiction